From 4ada3a5ce4aa4cfe7e3f0a066c2b9be7281f8930 Mon Sep 17 00:00:00 2001
From: Christian Kellner <christian@kellner.me>
Date: Wed, 6 Jun 2018 20:34:56 +0200
Subject: [PATCH] test: integration: use default python

Instead of explicitly specifying the python version, use the
default, i.e. /usr/bin/python
---
 tests/test-integration | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test-integration b/tests/test-integration
index d2fd7df..66fb89f 100755
--- a/tests/test-integration
+++ b/tests/test-integration
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 # -*- coding: utf-8 -*-
 #
 # bolt integration test suite
-- 
2.17.1

